src/mds/DamageTable.cc

```diff
@@ -200,6 +200,11 @@ bool DamageTable::notify_remote_damaged(inodeno_t ino, std::string_view path)
   return false;
 }
 
+damage_entry_id_t DamageTable::inode_to_id(inodeno_t ino)
```

Comment:
Does this handle cases when the damage (type) is a dirfrag or a dentry?

Reply:
It does, https://github.com/ceph/ceph/blob/main/src/mds/DamageTable.cc#L259-L264

Reply:
Can you explain how? `remotes` track damaged backtraces.

Reply:
> Can you explain how? `remotes` track damaged backtraces.

@vshankar It's not actually handling dirfrag, dentry. I guess @dparmar misunderstood. I am working on it.

Reply:
my bad `inode_to_id` doesnt handle all cases, misunderstood it this fn with `erase()`